VideoLocker/res/layout/row_course_outline_list.xml

```diff
@@ -4,8 +4,8 @@
     android:layout_width="match_parent"
     android:layout_height="64dp"
     android:paddingTop="4dp"
-    android:paddingBottom="4dp"
     android:orientation="vertical"
+    android:background="@drawable/course_list_item_selector"
```

Comment:
Hrrrm. This whole thing is kind of weird. CourseOutlineAdapter is the only subclass of CourseBaseAdapter and they seem pretty tied together, so it's not really clear to me why they're separate anyway. Do you think it would just make sense to collapse the whole thing? I'm almost always in favor of getting rid of inheritance relationships.